src/main/resources/net/snowflake/ingest/ingest_error_messages.properties

```diff
@@ -37,5 +37,6 @@
 0032=URI builder fail to build url: {0}
 0033=OAuth token refresh failure: {0}
 0034=Invalid config parameter: {0}
-0035=Too large batch of rows passed to insertRows, the batch size cannot exceed {0} bytes, recommended batch size for optimal performance and memory utilization is {1} bytes. We recommend splitting large batches into multiple smaller ones and call insertRows for each smaller batch separately.
-0036=Failed to load {0}. If you use FIPS, import BouncyCastleFipsProvider in the application: {1}
\ No newline at end of file
+0035=Failed to load {0}. If you use FIPS, import BouncyCastleFipsProvider in the application: {1}
+0036=Failed to drop channel: {0}
+0037=Deployment ID mismatch, Client was created on: {0}, Got upload location for: {1}. Please restart client: {2}.
```

Comment:
should this be `please recreate client?`

Reply:
I meant it as "shut down your application and restart it".